packages/base/src/Skeleton/Skeleton.tsx

```diff
@@ -38,6 +38,10 @@ export interface SkeletonProps extends ViewProps {
    * @type React Component
    */
   LinearGradientComponent?: React.ComponentType<any>;
+  /**
+   * Custom style for skeleton container
+   */
+  style: StyleProp<ViewStyle>;
```

Comment:
`style` is not needed, in my opinion, since `SkeletonProps` already extends `ViewProps`.

Reply:
Yes. Props Updated.